src/Service/PaymentReturnService.php

```diff
@@ -126,7 +126,7 @@ public function handlePaidStatus(Order $order, $transaction, $orderStatus, $paym
                 break;
             }
         }
-
+        $orderStatus = Mollie\Utility\OrderStatusUtility::transformPaymentStatusToRefunded($transaction);
```

Comment:
import OrderStatusUtility instead of using its namespace every time

Reply:
As @tomas862 said you cannot simply import object in main module class (cuz its breaking for some reason), but it is safe to do it everywhere else. Idk maybe that is why you avoid import anywhere (wasn't really aware of that) :man_shrugging:

Reply:
on services and controllers it can be no brainer to import them - we should only avoid import in main module class